Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -121,119 +121,214 @@ uint64_t AODProducerWorkflowDPL::getTFNumber(uint64_t firstVtxGlBC, int runNumbe
-                    (Char_t)(128. * track.getSigma1PtY() / track.getSigma1Pt2() / track.getSigmaY2()),
-                    (Char_t)(128. * track.getSigma1PtZ() / track.getSigma1Pt2() / track.getSigmaZ2()),
-                    (Char_t)(128. * track.getSigma1PtSnp() / track.getSigma1Pt2() / track.getSigmaSnp2()),
-                    (Char_t)(128. * track.getSigma1PtTgl() / track.getSigma1Pt2() / track.getSigmaTgl2()));
-    tracksExtraCursor(0,
-                      truncateFloatFraction(tpcInnerParam, mTrack1Pt),
-                      flags,
-                      itsClusterMap,
-                      tpcNClsFindable,
-                      tpcNClsFindableMinusFound,
-                      tpcNClsFindableMinusCrossedRows,
-                      tpcNClsShared,
-                      trdPattern,
-                      truncateFloatFraction(itsChi2NCl, mTrackCovOffDiag),
-                      truncateFloatFraction(tpcChi2NCl, mTrackCovOffDiag),
-                      truncateFloatFraction(trdChi2, mTrackCovOffDiag),
-                      truncateFloatFraction(tofChi2, mTrackCovOffDiag),
-                      truncateFloatFraction(tpcSignal, mTrackSignal),
-                      truncateFloatFraction(trdSignal, mTrackSignal),
-                      truncateFloatFraction(tofSignal, mTrackSignal),
-                      truncateFloatFraction(length, mTrackSignal),
-                      truncateFloatFraction(tofExpMom, mTrack1Pt),
-                      truncateFloatFraction(trackEtaEMCAL, mTrackPosEMCAL),
-                      truncateFloatFraction(trackPhiEMCAL, mTrackPosEMCAL));
-  }
-}
-
-template <typename TTracks, typename TTracksCursor>
-void AODProducerWorkflowDPL::fillMFTTracksTable(const TTracks& tracks, std::vector<int>& vCollRefs, const TTracksCursor& mftTracksCursor)
-{
-  for (int i = 0; i < tracks.size(); i++) {
-    auto& track = tracks[i];
-    int collisionID = vCollRefs[i];
-
-    mftTracksCursor(0,
-                    collisionID,
-                    track.getX(),
-                    track.getY(),
-                    track.getZ(),
-                    track.getPhi(),
-                    track.getTanl(),
-                    track.getInvQPt(),
-                    track.getNumberOfPoints(),
-                    track.getTrackChi2());
+  int collisionID;
+  int trackType;
+  float tpcInnerParam = 0.f;
+  uint32_t flags = 0;
+  uint8_t itsClusterMap = 0;
+  uint8_t tpcNClsFindable = 0;
+  int8_t tpcNClsFindableMinusFound = 0;
+  int8_t tpcNClsFindableMinusCrossedRows = 0;
+  uint8_t tpcNClsShared = 0;
+  uint8_t trdPattern = 0;
+  float itsChi2NCl = -999.f;
+  float tpcChi2NCl = -999.f;
+  float trdChi2 = -999.f;
+  float tofChi2 = -999.f;
+  float tpcSignal = -999.f;
+  float trdSignal = -999.f;
+  float tofSignal = -999.f;
+  float length = -999.f;
+  float tofExpMom = -999.f;
+  float trackEtaEMCAL = -999.f;
+  float trackPhiEMCAL = -999.f;
+
+  for (int i = 0; i < vTracksIdx.size(); i++) {
```

Comment:
Might be my ignorance, but how the tracks redundancy (the fact that e.g. ITS tracks is already accounted in the ITS-TPC) are accounted.I don't see the references of "more global" tracks to contributing simpler tracks.

Reply:
Yes, redundant tracks are not accounted for in this code. Same as in previous comment: I think I need RecoContainer for this to get only more 'global' tracks. So I can implement it after #6170 is merged, where I added RecoContainer to the producer.